src/api/app/components/bs_request_activity_timeline_component.rb

```diff
@@ -23,9 +23,9 @@ def initialize(bs_request:)
   end
 
   def diff_for_ref(comment)
-    return unless comment.diff_ref
+    return unless (ref = comment.diff_ref&.match(/diff_([0-9]*)/))
```

Comment:
```suggestion
    return unless (ref = comment.diff_ref&.match(/diff_([0-9]+)/))
```
We can replace here the `*` by the `+`. We don't need to take in account the case of not having a digit.